patches/common/chromium/chrome_process_finder.patch

```diff
@@ -0,0 +1,46 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <miburda@microsoft.com>
+Date: Fri, 26 Oct 2018 20:45:49 +0200
+Subject: chrome_process_finder.patch
+
+Pass original command line
+
+diff --git a/chrome/browser/win/chrome_process_finder.cc b/chrome/browser/win/chrome_process_finder.cc
+index 4fdea2af27020303a66a747304c5ecce5cc698d5..b25c45a61f50d244d10283bd075b722d75ba98b5 100644
+--- a/chrome/browser/win/chrome_process_finder.cc
++++ b/chrome/browser/win/chrome_process_finder.cc
+@@ -19,9 +19,6 @@
+ #include "base/win/scoped_handle.h"
+ #include "base/win/win_util.h"
+ #include "base/win/windows_version.h"
+-#include "chrome/common/chrome_constants.h"
```

Comment:
We do build `//chrome/common` as a dep,  so it need not be removed.

Reply:
restored